docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+
+Management::
+* Adds timezone support for Transforms date histogram pivot configuration {kibana-pull}155535[#155535]
+* Adds more system indices for storing internal data {kibana-pull}154888[#154888]
+* Adds option to Reauthorize transform in Management page {kibana-pull}154736[#154736]
+* Adds secondary authorization header to Transforms in Fleet {kibana-pull}154665[#154665]
+* Adds improvements for supporting counter fields in Transforms {kibana-pull}154171[#154171]
+* Adds `_schedule_now` action to transform list {kibana-pull}153545[#153545]
+* Adds link from Index management {kibana-pull}152640[#152640]
+* Adds health information for alerting rule in Transforms{kibana-pull}152561[#152561]
+* Adds improvements for index pattern input in the data view flyout {kibana-pull}152138[#152138]
+* Adds a new description for the metadata field in ingest pipelines {kibana-pull}150935[#150935]
+* Adds a _meta field to the Ingest pipelines form {kibana-pull}149976[#149976]
+
+Maps::
+Adds metrics mask {kibana-pull}154983[#154983]
+
+Observability::
+* Adds invalid license page {kibana-pull}154866[#154866]
+* Adds empty state page links {kibana-pull}154678[#154678]
+* Adds upload symbols instructions to add data page {kibana-pull}154670[#154670]
+* Adds new CPU incl and CPU excl names {kibana-pull}154560[#154560]
+* Adds symbols callout on frame information window {kibana-pull}154478[#154478]
+* Adds Co2 and dollar cost columns and show more information action to functions table {kibana-pull}154097[#154097]
+* Adds improvements to functions {kibana-pull}153873[#153873]
+* Adds improvements to Flamegraph {kibana-pull}153598[#153598]
+* Adds the ability to open the Traces view when you click on a series in stacked charts {kibana-pull}153325[#153325]
+* Adds CPU usage column to replace CPU count column {kibana-pull}151696[#151696]
+
+Querying & Filtering::
+* Adds the ability to avoid duplicate host IP mapping {kibana-pull}155353[#155353]
+* Adds improvements to the saved query terminology {kibana-pull}154517[#154517]
+
+Security::
+* Adds the ability to configure remote index privileges directly from Role management {kibana-pull}154948[#154948]
+* Adds the Cloud_defend process index as a source and merged process event handling {kibana-pull}153213[#153213]
+
+[float]
+[[fixes-v8.8.0]]
+=== Bug Fixes
+Alerting::
+* Fixes Delete Schedule button padding issue {kibana-pull}154503[#154503]
+* Fixes error message flash and throttle value reset {kibana-pull}154497[#154497]
+* Fixes broken custom snooze recurrences with monthly frequency {kibana-pull}154251[#154251]
+* Fixes an issue where you were unable to use retry on updateAPIKey conflict {kibana-pull}151802[#151802]
+
+APM::
+* Fixes an issue where you were uneable to enable framework alerts as data by default {kibana-pull}154076[#154076]
+* Upgraded EUI to v76.0.0 {kibana-pull}152506[#152506]
+* Fixes an issue where the OpenTelemetry process and system metrics were unsupported {kibana-pull}151826[#151826]
+
```

Comment:
Could you please add this PR https://github.com/elastic/kibana/pull/155897 in a `Cases` section?